orthanc/orthanc-anon/plugin/pixl.py

```diff
@@ -156,6 +157,34 @@ def SendViaStow(resourceId):
         orthanc.LogError("Failed to send via STOW")
 
 
+def SendViaFTPS(resourceId: str) -> None:
+    """
+    Makes a POST API call to upload the resource to a dicom-web server
+    using orthanc credentials as authorisation
+    """
+    ORTHANC_USERNAME = config("ORTHANC_USERNAME")
+    ORTHANC_PASSWORD = config("ORTHANC_PASSWORD")
+
```

Comment:
probably would be good to know this doens't exist when we load up DICOM server rather than at export, so making this in the top level `pixl.py` file might be nicer?